File: examples/python/search_vehicles.py
from pprint import pprint
from apibara_client import apibara_get

vehicles = apibara_get("/vehicles", {
    "make": "Toyota",
    "model": "Camry",
    "year_from": 2018,
    "lot_sub_status": "Open",
    "per_page": 20,
})

pprint(vehicles)

File: examples/php/search-vehicles.php
<?php
require __DIR__ . '/ApibaraClient.php';

$client = new ApibaraClient(getenv('APIBARA_API_KEY'));

$data = $client->get('/vehicles', [
    'make' => 'Toyota',
    'model' => 'Camry',
    'year_from' => 2018,
    'lot_sub_status' => 'Open',
    'per_page' => 20,
]);

echo json_encode($data, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES) . PHP_EOL;
